lsstswBuild.sh

```diff
@@ -63,7 +63,7 @@ print_error() {
 }
 
 fail() {
-    code=${2:1}
+    local code=${2:-1}
```

Comment:
Really just an issue of consistent style in declaring all param variables as local.